### ISSUE TITLE
File to import not found

### ISSUE THREAD

**Issue by andidev** [MoOx/compass-recipes#51]
The import in stylesheets/recipes/form/_skin.scss (stable branch) is 

```
@import "recipes/form/skin/default";
```

...when it should be 

```
@import "recipes/form/skin/natural";
```

...or the change the name of _natural.scss to _default.scss

**Comment by MoOx** [comment]
I'll fix this tomorrow & release it with #52
But I encourage you to switch to 3 alpha (alpha because of sass alpha dependency - but personally I use this version everyday)

**Comment by andidev** [comment]
Okey I'll do so, thanks for the tip and for some great recipes!
